featuretools/primitives/base/aggregation_primitive_base.py

```diff
@@ -7,6 +7,17 @@ class AggregationPrimitive(PrimitiveBase):
     base_of = None  # whitelist of primitives this prim can be input for
     base_of_exclude = None  # primitives this primitive can't be input for
     stack_on_self = True  # whether or not it can be in input_types of self
+    has_series_library = False
+    computed_has_series_library = False
+
+    @classmethod
+    def uses_series_library_argument(cls):
```

Comment:
I think checking for `agg_type` (and changing the name of the method accordingly) would be more beneficial because we could raise a `DeprecationWarning` if we find that `agg_type` is used, whereas if we find that `series_library` is not used we still don't know if `agg_type` is present or not defined

Reply:
Should we raise `DeprecationWarning` once per primitive?